src/components/form-slice/FormPartBasic.tsx

```diff
@@ -5,25 +5,32 @@ import { FormItemTextInput } from '../form/TextInput';
 import { FormItemTextarea } from '../form/Textarea';
 import { useFormContext } from 'react-hook-form';
 import type { APISIXType } from '@/types/schema/apisix';
+import { useNamePrefix } from '@/utils/useNamePrefix';
+import type { PropsWithChildren } from 'react';
 
-export type FormPartBasicProps = Omit<FormSectionProps, 'form'>;
+export type FormPartBasicProps = Omit<FormSectionProps, 'form'> &
+  PropsWithChildren;
 
 export const FormPartBasic = (props: FormPartBasicProps) => {
+  const { children, ...restProps } = props;
   const { control } = useFormContext<APISIXType['Basic']>();
   const { t } = useTranslation();
+  const np = useNamePrefix();
+
   return (
-    <FormSection legend={t('form.basic.title')} {...props}>
+    <FormSection legend={t('form.basic.title')} {...restProps}>
       <FormItemTextInput
-        name="name"
+        name={np('name')}
         label={t('form.basic.name')}
         control={control}
       />
       <FormItemTextarea
-        name="desc"
+        name={np('desc')}
         label={t('form.basic.desc')}
         control={control}
       />
-      <FormItemLabels name="labels" control={control} />
+      <FormItemLabels name={np('labels')} control={control} />
```

Comment:
<img width="931" alt="image" src="https://github.com/user-attachments/assets/0be69af3-da03-426e-bb86-b6ac172c38a9" />


I just tried it. I can only make it work by inputting `,` or by blurring it.